src/index.tsx

```diff
@@ -403,8 +423,35 @@ export function useMicrophone() {
     },
     []
   );
+  /**
+   * Starts local voice activity detection.
+   * @param config configuration of the voice activity monitor
+   */
+  const startVADMonitor = useCallback(
+    (
+      config: VADConfiguration = { monitorInterval: 250, samplingRate: 22050 }
+    ) => {
+      RNSoundLevel.start(config);
+      setIsVADOn(true);
+    },
+    []
+  );
+  /**
+   * Stop local voice activity detection.
+   */
+  const stopVADMonitor = useCallback(() => {
+    RNSoundLevel.stop();
+    setIsVADOn(false);
+  }, []);
 
-  return { isMicrophoneOn, toggleMicrophone, startMicrophone };
+  return {
+    isMicrophoneOn,
+    isSpeaking,
+    toggleMicrophone,
+    startVADMonitor,
```

Comment:
please add it also in the example app, maybe in dev mode (long press on membrane logo)